packages/client/src/hooks/useAppDispatch/index.ts

```diff
@@ -0,0 +1,6 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux'
+import { AppDispatch, RootState } from '../../store/types'
+
+export const useAppDispatch = (): AppDispatch => useDispatch<AppDispatch>()
```

Comment:
```suggestion
export const useAppDispatch = (): AppDispatch => useDispatch
```

см https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks